atlas/portal/api/portal.api.proto

```diff
@@ -0,0 +1,126 @@
+syntax = "proto3";
+package portal.pb;
+option go_package = "atlas/portal/pkg/pb";
+
+import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
+
+service Portal{
+  rpc GetVersion (google.protobuf.Empty) returns (VersionResponse) {
+    option (google.api.http) = {
+      get: "/version"
+    };
+  }
+
+  rpc GetInfo  (GetInfoRequest) returns (GetInfoResponse) {
+    option (google.api.http) = {
+      post: "/getInfo"
+      body: "*"
+    };
+  }
+
+  rpc SetInfo  (SetInfoRequest) returns (SetInfoResponse) {
+    option (google.api.http) = {
+      post: "/setInfo"
+      body: "*"
+    };
+  }
+
+  rpc GetUptime (GetUptimeRequest) returns (GetUptimeResponse) {
+    option (google.api.http) = {
+      post: "/getUptime"
```

Comment:
Action **get**Uptime, which uses **POST** method, looks a bit confusing. Please avoid of such things

Reply:
Done